### PR TITLE
chore: delete duplicated person record

### PR DESCRIPTION
**Pull request by JosephKelly**
Data recorded against tis id: 265154

TIS21-4266: Merge Trainee Record

### CODE REVIEW

File: tcs-service/src/main/resources/db/migration/data/hee/V4.24__delete_duplicatedPersonRecord.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+delete from RightToWork where id in(305827);
```
Comment (jayanta2018) [inline tcs-service/src/main/resources/db/migration/data/hee/V4.24__delete_duplicatedPersonRecord.sql:1]: Just wondering is the person id is `305827` or 265154? I think in the app you shared the page with the person id 265154.

Comment (JosephKelly) [reply tcs-service/src/main/resources/db/migration/data/hee/V4.24__delete_duplicatedPersonRecord.sql:1]: Yep so the one which will remain is `265154` which has more 'actual' data recorded against it

**Comment by hihilary** [comment]
Just had a quick check on the UI. There're more meaningful data under Sensitive data/DIVERSITY. Do we want to copy them over?

**Comment by JosephKelly** [comment]
> Just had a quick check on the UI. There're more meaningful data under "Sensitive data/DIVERSITY" section. Do we want to copy them over?

Thanks for the further check... I'll let the reporter know on Teams & ask them to make sure.

**Pull request merged**